packages/my-account/src/components/composite/Subscription/SubscriptionNextRunProgress/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { OrderSubscription } from "@commercelayer/sdk"
+import { formatDistanceStrict } from "date-fns"
+import { useTranslation } from "react-i18next"
+import { ProgressTitle } from "./styled"
+
+interface Props {
+  subscription: OrderSubscription
+}
+
+type SubscriptionFrequency = 'hourly' | 'daily' | 'weekly' | 'monthly' | 'two-month' | 'three-month' | 'four-month' | 'six-month' | 'yearly'
```

Comment:
frequency can also be a crontab expression

Reply:
Removed this one! No more needed after last code cleanup